hack/boilerplate/boilerplate.go.txt

```diff
@@ -0,0 +1 @@
+// Copyright (c) YEAR Tigera, Inc. All rights reserved.
```

Comment:
should this be checked in?

Reply:
Yeah we need this file for `make gen-datastore-bindings`.

Reply:
Should replace YEAR with 2020, if you still need this file after what we discussed earlier.

Reply:
No longer needed.